src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.scss

```diff
@@ -37,14 +37,82 @@ $backgroundColor: $black;
 
 .geneViewTabs {
   grid-area: 2 / 1 / 3 / 4;
+  display: grid;
+  grid-template-areas:
+                  "filter-toggle tabs"
+                  "filters filters";
+  grid-template-columns: 170px auto;
+  grid-template-rows: auto auto;
   z-index: 2;
   position: sticky;
   top: -80px;
-  height: 100%;
+  height: 100%;  
+  background-color: $backgroundColor;
+}
+
+.tabWrapper {
+  grid-area: tabs;
+  height: 76px;
+  display: flex;
+  align-items: flex-end;
+  padding: 4px;
   background-color: $backgroundColor;
-  padding-left: 180px;
+}
+
+.filtersWrapper {
+  grid-area: filters;
 }
 
 .geneViewTabContent {
   grid-area: 3 / 1 / 4 / 4;
 }
+
+.filterLabelContainer {
+  grid-area: filter-toggle;
+  align-self: end;
+  padding-right: 20px;
+  padding-bottom: 9px;
+  padding-top: 6px;
+  position: sticky;
+  top: -37px;  
```

Comment:
Do we need this? I removed this in developer tools and didn't notice any difference without it.

Reply:
That's needed when scrolling on the page to make it fixed to the top of the page.

Reply:
Okay let's leave it like that then